lib/src/utils/app_routes.dart

```diff
@@ -1,13 +1,31 @@
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 import 'package:mustang_viewer/src/screens/connect/connect_screen.dart';
+import 'package:mustang_viewer/src/screens/difference/difference_screen.dart';
 import 'package:mustang_viewer/src/screens/memory/memory_screen.dart';
 
 class AppRoutes {
   static const String connect = '/connect';
   static const String memory = '/memory';
+  static const String difference = '/difference';
 
-  static final Map<String, WidgetBuilder> routeBuilders = {
-    AppRoutes.connect: (BuildContext _) => const ConnectScreen(),
-    AppRoutes.memory: (BuildContext _) => const MemoryScreen(),
-  };
+  static Route? onGenerateRoute(RouteSettings routeSettings) {
+    switch (routeSettings.name) {
+      case AppRoutes.difference:
+        return PageRouteBuilder(
+          pageBuilder: (_, __, ___) => const DifferenceScreen(),
+          transitionDuration: Duration.zero,
+        );
+      case AppRoutes.memory:
+        return PageRouteBuilder(
```

Comment:
Why not `https://api.flutter.dev/flutter/material/MaterialPageRoute-class.html`  instead of `PageRouteBuilder` ?

Reply:
I tried `MaterialPageRoute` but it doesn't let me control the transition animations.